sdk/cosmos/azure_data_cosmos/Cargo.toml

```diff
@@ -42,6 +44,7 @@ default = ["hmac_rust"]
 hmac_rust = ["azure_core/hmac_rust"]
 hmac_openssl = ["azure_core/hmac_openssl"]
 key_auth = [] # Enables support for key-based authentication (Primary Keys and Resource Tokens)
+test_e2e = ["key_auth"]
```

Comment:
Could we phase this out? I got rid of it in Event Hubs because we can't do `--all-features` to test, which left many tests in a failed state as you may recall. You can check the `TestContext::test_mode()` accessor to see if you're running "live", for example, if that helps.

Reply:
This can actually be removed, I just forgot to.